tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -0,0 +1,227 @@
+      -vv
+    #elif $verbosity == 3
+      -vvv
+    #end if
+    ${" ".join(map(str, $input_fastas))}
+  ]]></command>
+  <inputs>
+    <param 
+      name="input_fastas" 
+      type="data" 
+      label="Input Genome(s)" 
+      optional="false" 
+      multiple="true"
+      format="fasta"
+      />
+    <param 
+      name="output_format" 
+      type="select" 
+      label="Results output format"
+      multiple="false" >
+      <option value="csv" selected="true">
+        CSV (Comma Separated Values)
+      </option>
+      <option value="json">
+        JSON (JavaScript Object Notation)
+      </option>
+    </param>
+    <param 
+      name="use_full_cgmlst_db"
+      type="boolean"
+      checked="false"
+      label="Use full cgMLST database for serovar prediction. About 10X slower with equivalent results to reduced centroid allele database."
+      />
+    <param 
+      name="run_mash"
+      type="boolean"
+      checked="true"
+      label="Run Mash MinHash-based serovar prediction"
+      />
+    <param 
+      name="no_cgmlst"
+      type="boolean"
+      checked="false"
+      label="Skip running cgMLST-based serovar prediction"
+      />
+    <param 
+      name="qc"
+      type="boolean"
+      checked="true"
+      label="Basic QC of results"
+      />
+    <param
+      name="tmp_dir"
+      type="text"
+      value="/tmp"
+      label="Temporary analysis directory"
+     />
+    <param 
+      name="keep_tmp"
+      type="boolean"
+      checked="false"
+      label="Keep temporary analysis directory"
+      />
+    <param 
+      name="threads"
+      type="integer"
+      value="1"
+      label="Number of threads (1 genome will only use 1 thread)"
+      />
+    <param 
+      name="verbosity"
+      type="select" 
+      label="Logging verbosity">
+      <option value="0">
+        Error messages only
+      </option>
+      <option value="1">
+        Show warning messages
+      </option>
+      <option value="2" selected="true">
+        Show info messages
+      </option>
+      <option value="3">
+        Show debug messages
+      </option>
+    </param>
+  </inputs>
+  <outputs>
+    <!-- TODO: Fix genome fasta name mangling by Galaxy; try to use original name if possible -->
+    <data 
+      name="output_prediction_csv" 
+      format="tabular" 
+      label="SISTR Results"
+      from_work_dir="sistr-report.csv">
+      <filter>output_format == "csv"</filter>
+    </data>
+    <data 
+      name="output_prediction_json" 
+      format="json" 
+      label="SISTR Results"
+      from_work_dir="sistr-report.json">
+      <filter>output_format == "json"</filter>
+    </data>
+    <data 
+      name="cgmlst_profiles" 
+      format="tabular" 
+      label="cgMLST results" />
+    <data
+      name="novel_alleles"
+      format="fasta" 
+      label="Novel cgMLST alleles" />
+    <data 
+      name="alleles_output"
+      format="json"
+      label="cgMLST allele match results" />
+  </outputs>
+  <tests>
```

Comment:
I've added 2 tests with a complete Salmonella Paratyphi B genome and the partial genome included in ``sistr-error.zip`` below in 8cf4638.